lib/kit.js

```diff
@@ -0,0 +1,31 @@
+let instance, formatter;
+
+if (process.env.NO_COLOR)
+  instance = require('./logger/simple.js');
+else
+  instance = require('./logger/rainbow.js');
+
+formatter = (message, timestamp = new Date().toString()) => {
```

Comment:
I would personally try to keep the timestamp as short as possible, good first step would be

![image](https://user-images.githubusercontent.com/546845/38420052-dd3c6db2-39a2-11e8-9aab-863298d7360f.png)